### PR TITLE
BCTHEME-958: Translation updates November 2021

### PR DESCRIPTION
**Pull request by bc-yevhenii-buliuk**
#### What?
This PR updates lang-files and schemaTranslations.json for the Cornerstone theme.

#### Tickets / Documentation
- [BCTHEME-958](https://jira.bigcommerce.com/browse/BCTHEME-958)

#### Screenshots
958_en
958_de

### CODE REVIEW

**Review by bc-vlad-dlogush — APPROVED** [review]

**Review by BC-krasnoshapka — APPROVED** [review]

File: lang/es-MX.json [hunk 1]
```diff
@@ -0,0 +1,1011 @@
...
+        "unsaved_changes": "Tiene cambios sin guardar",
+        "by_using": "Al utilizar nuestro sitio web, acepta",
+        "agree_on_data_collection": "Al utilizar nuestro sitio web, acepta la recopilación de datos tal y como se describe en ",
+        "change_preferences": "Puede cambiar sus preferencias en cualquier momento",
+        "cancel_dialog_title": "¿Está seguro de que desea cancelar?",
+        "privacy_policy": "Política de privacidad",
+        "allow_category_tracking": "Permitir el seguimiento de [CATEGORY_NAME]",
+        "disallow_category_tracking": "No permitir el seguimiento de [CATEGORY_NAME]"
+    }
+}
```
Comment (yurytut1993) [inline lang/es-MX.json:1011]: Can you please add newline to the end of this and other files according to github suggestion?

File: lang/fr.json [hunk 2]
```diff
@@ -913,7 +913,7 @@
         "play_pause_button_pause": "Pause",
         "play_pause_button_aria_play": "Lancer le carrousel",
         "play_pause_button_aria_pause": "Arrêter le carrousel",
-        "slide_number": "Diapositive n° {slide_number}"
+        "slide_number": "Diapositive n°  {slide_number}"
```
Comment (yurytut1993) [inline lang/fr.json:916]: Looks like we have extra space here

File: lang/es-MX.json (hunk 1 above)
Comment (bc-yevhenii-buliuk) [reply lang/es-MX.json]: @yurytut1993 done!

File: lang/fr.json (hunk 2 above)
Comment (bc-yevhenii-buliuk) [reply lang/fr.json]: @yurytut1993 done!

**Review by yurytut1993 — APPROVED** [review]
Nice work

**Review by bc-alexsaiannyi — APPROVED** [review]
Looks good. Just fix a failing check. It seems that we have unused translation key `i18n.AdminBar`

**Review by bc-azvierieva — APPROVED** [review]

**Pull request merged**